src/pkg/services/m365/m365.go

```diff
@@ -313,6 +313,82 @@ func SitesMap(
 	return idname.NewCache(itn), nil
 }
 
+// ---------------------------------------------------------------------------
+// Teams
+// ---------------------------------------------------------------------------
+
+// Team is the minimal information required to identify and display a M365 Team.
+type Team struct {
+	ID string
+
+	// DisplayName is the human-readable name of the team.  Normally the plaintext name that the
+	// user provided when they created the team or the updated name if it was changed.
+	// Ex: displayName: "Testing Team"
+	DisplayName string
+}
+
+// TeamsCompat returns a list of teams in the specified M365 tenant.
+func TeamsCompat(ctx context.Context, acct account.Account) ([]*Team, error) {
+	errs := fault.New(true)
+
+	us, err := Teams(ctx, acct, errs)
+	if err != nil {
+		return nil, err
+	}
+
+	return us, errs.Failure()
+}
+
+// Teams returns a list of teams in the specified M365 tenant
+func Teams(ctx context.Context, acct account.Account, errs *fault.Bus) ([]*Team, error) {
+	ac, err := makeAC(ctx, acct, path.GroupsService)
+	if err != nil {
+		return nil, clues.Stack(err).WithClues(ctx)
+	}
+
+	return getAllTeams(ctx, ac.Groups())
+}
+
+// parseUser extracts information from `models.Userable` we care about
+func parseTeam(item models.Groupable) (*Team, error) {
+	if item.GetDisplayName() == nil {
+		return nil, clues.New("Team missing display name").
```

Comment:
Golang standards: errors should not begin with capitol letters.
```suggestion
		return nil, clues.New("group missing display name").
```